README.md

```diff
@@ -1,2 +1,33 @@
 # fluent-bit-multiline
+
 Fast and Lightweight Logs and Metrics processor for Linux, BSD, OSX and Windows.
+
+The main goal of this project is to extend the default aws-for-fluent-bit docker image
+to support `partial_message` mode of fluent bit. Partial/Split container logs are then concatenated into one log.
+This is useful if the logging destination expects logs to be single line or if you
+log in JSON format (for ex. to Datadog or OpenSearch).
+
+For more information, see here: [Example from AWS](https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/filter-multiline-partial-message-mode)
+
+The config also configures the container to provide an endpoint for healthcheck data.
+Go to the [FluentBit Documentation](https://docs.fluentbit.io/manual/administration/monitoring#health-check-for-fluent-bit) for more information.
+
+## Usage
+
+Use this image inside of container definitions for AWS ECS Services. Make sure to reference the custom configuration
+file! Example:
+
+```json
+{
+  "image": "ghcr.io/hapag-lloyd/fluent-bit-multiline:edge",
```

Comment:
issue: we should refer to a specific version, shouldn't we?

Reply:
Unsure, since version tags must be updated every time anything changes. Not sure how easy that can be automated. If not automated, it will definitely be forgotten. Therefore I decided that referring to the `edge` tag is good enough here, since it automatically points to the newest version.

IMO `edge` also signifies clearly that it is sketchy to use if you need maximum stability and repeatability.

What are your thoughts on this?

Reply:
Maybe we are fine with a comment. We will always use the exact version and ask Renovate for updates.

Reply:
This would cause an endless "update" loop, especially if commit hashes are used as docker tag.
Updating the version after a PR is merged always causes the problem of update loops unless the Readme update does not trigger releases. Then however we allow `main` to not be built occasionally which may be fine but feels wrong.

Updating the version pre-merge is difficult since the actual version the merge is included in is not known before the merge.

I can change the docs to specifically point out the danger of using tags like `latest` or `edge` but automatically updating the version seems over-engineered at this point.

Reply:
> I can change the docs to specifically point out the danger of using tags like latest or edge
Great.